net/socket/getsockopt.c

```diff
@@ -271,6 +264,13 @@ int psock_getsockopt(FAR struct socket *psock, int level, int option,
 {
   int ret = -ENOPROTOOPT;
 
+  /* Verify that the socket option if valid (but might not be supported ) */
```

Comment:
I can't verify the code change itself, but found one tiny typo:

```suggestion
  /* Verify that the socket option is valid (but might not be supported ) */
```

Reply:
> I can't verify the code change itself, but found one tiny typo:

done.